--- a/source/core/gameconfigfile.cpp	2021-04-18 02:31:51.992515000 +0100
+++ b/source/core/gameconfigfile.cpp	2021-04-18 02:37:48.675277965 +0100
@@ -99,19 +99,26 @@
 		// Arch Linux likes them in /usr/share/raze
 		// Debian likes them in /usr/share/games/raze
 		// I assume other distributions don't do anything radically different
-		SetValueForKey ("Path", "/opt/raze", true);
-		SetValueForKey ("Path", "/usr/share/games/raze", true);
-		SetValueForKey ("Path", "/usr/local/share/games/raze", true);
-		SetValueForKey ("Path", "/usr/share/games/jfduke3d", true);
-		SetValueForKey ("Path", "/usr/local/share/games/jfduke3d", true);
-		SetValueForKey ("Path", "/usr/share/games/eduke32", true);
-		SetValueForKey ("Path", "/usr/local/share/games/eduke32", true);
-		SetValueForKey ("Path", "/usr/share/games/nblood", true);
-		SetValueForKey ("Path", "/usr/local/share/games/nblood", true);
-		SetValueForKey("Path", "/usr/share/games/jfsw", true);
-		SetValueForKey("Path", "/usr/local/share/games/jfsw", true);
-		SetValueForKey("Path", "/usr/share/games/voidsw", true);
-		SetValueForKey("Path", "/usr/local/share/games/voidsw", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/blood", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/blood/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/duke3d", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/duke3d/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/exhumed", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/exhumed/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/nam", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/nam/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneck", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneck/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneckrides", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneckrides/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneckdeer", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneckdeer/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/shadow", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/shadow/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/ww2gi", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/ww2gi/*", true);
+		SetValueForKey("Path", "/usr/share/raze", true);
+		SetValueForKey("Path", "/usr/share/games/raze", true);
 
 #endif
 		SetValueForKey ("Path", "$STEAM", true); // also covers GOG.
@@ -131,13 +138,26 @@
 		SetValueForKey ("Path", "$GAMEDIR", true);
 #else
 		SetValueForKey ("Path", "$HOME/" GAME_DIR, true);
-		SetValueForKey ("Path", SHARE_DIR, true);
-		SetValueForKey ("Path", "/usr/share/games/jfduke3d", true);
-		SetValueForKey ("Path", "/usr/local/share/games/jfduke3d", true);
-		SetValueForKey ("Path", "/usr/share/games/eduke32", true);
-		SetValueForKey ("Path", "/usr/local/share/games/eduke32", true);
-		SetValueForKey ("Path", "/usr/share/games/nblood", true);
-		SetValueForKey ("Path", "/usr/local/share/games/nblood", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/blood", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/blood/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/duke3d", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/duke3d/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/exhumed", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/exhumed/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/nam", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/nam/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneck", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneck/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneckrides", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneckrides/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneckdeer", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/redneckdeer/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/shadow", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/shadow/*", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/ww2gi", true);
+		SetValueForKey("Path", "$HOME/Arch-RGS/roms/ports/raze/ww2gi/*", true);
+		SetValueForKey("Path", "/usr/share/raze", true);
+		SetValueForKey("Path", "/usr/share/games/raze", true);
 #endif
 	}
 
@@ -154,10 +174,7 @@
 		SetValueForKey("Path", "$PROGDIR/soundfonts", true);
 #else
 		SetValueForKey("Path", "$HOME/" GAME_DIR "/soundfonts", true);
-		SetValueForKey("Path", "/usr/local/share/" GAME_DIR "/soundfonts", true);
-		SetValueForKey("Path", "/usr/local/share/games/" GAME_DIR "/soundfonts", true);
-		SetValueForKey("Path", "/usr/share/" GAME_DIR "/soundfonts", true);
-		SetValueForKey("Path", "/usr/share/games/" GAME_DIR "/soundfonts", true);
+		SetValueForKey("Path", "/usr/share/soundfonts", true);
 #endif
 	}
 
