--- a/nall/GNUmakefile	2020-10-07 18:15:04.000000000 
+++ b/nall/GNUmakefile	2020-10-07 18:53:00.000000000 
@@ -124,13 +124,14 @@
   flags   += -stdlib=libc++
   options += -lc++ -lobjc
 endif
 
 # linux settings
 ifeq ($(platform),linux)
-  options += -ldl
+  flags += ${CXXFLAGS}
+  options += ${LDFLAGS} -ldl
 endif
 
 # bsd settings
 ifeq ($(platform),bsd)
   flags   += -I/usr/local/include
   options += -Wl,-rpath=/usr/local/lib
