diff -u a/retroarch.c b/retroarch.c
--- a/retroarch.c	2021-03-28 21:43:18.000000000 +0100
+++ b/retroarch.c	2021-04-11 14:26:23.460259454 +0100
@@ -2552,7 +2552,8 @@
                   entry->label, entry->type, i, entry->entry_idx);
          break;
       case MENU_ACTION_SEARCH:
-         menu_input_dialog_start_search();
+         // disable search action
+         // menu_input_dialog_start_search();
          break;
       case MENU_ACTION_SCAN:
          if (cbs && cbs->action_scan)
